### PR TITLE
feat: additional loader configurations and re-organization

### PR DESCRIPTION
**Pull request by cofin**
## Description

-

### CODE REVIEW

File: advanced_alchemy/mixins/audit.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from datetime import datetime, timezone
+
+from sqlalchemy.orm import Mapped, declarative_mixin, mapped_column, validates
+
+from advanced_alchemy.types import DateTimeUTC
+
+
+@declarative_mixin
```
Comment (Alc-Alc) [inline advanced_alchemy/mixins/audit.py:10]: why was this added for all mixins? AFAIK this is not needed in 2.0

Comment (cofin) [reply advanced_alchemy/mixins/audit.py:10]: I added these to be consistent across all mixins.  We had one with them and the rest without.

We can remove them all of none of them are needed; but I'd just like it to be consistent.
